sdk/python/kfp/dsl/_metadata.py

```diff
@@ -221,6 +227,14 @@ def _extract_pipeline_metadata(func):
     arg_default = arg_defaults[arg] if arg in arg_defaults else None
     if arg in annotations:
       arg_type = _annotation_to_typemeta(annotations[arg])
+    if 'openapi_schema_validator' in arg_type.properties and arg_default is not None:
```

Comment:
`arg_default` may be an instance of `PipelineParam`.
https://github.com/kubeflow/pipelines/pull/1380 fixes that. Can we merge it?

Reply:
Sure.